packages/dto/src/comment.rs

```diff
@@ -1,34 +1,43 @@
 #![allow(unused_variables, unused)]
-use crate::Result;
+use crate::{Result, User};
 #[cfg(feature = "server")]
 use by_axum::aide;
 use by_macros::api_model;
 use by_types::QueryResponse;
 use validator::ValidationError;
 
-#[api_model(base = "/v1/topics/:topic-id/comments", table = comments, iter_type = QueryResponse)]
+#[api_model(base = "/v1/topics/:topic-id/comments", table = comments, action_by_id = like, iter_type = QueryResponse)]
 pub struct Comment {
     #[api_model(summary, primary_key, read_action = [get_comment, find_by_id])]
     pub id: i64,
     #[api_model(summary, auto = insert)]
     pub created_at: u64,
 
-    #[api_model(summary)]
-    pub profile_url: String,
+    #[api_model(many_to_one = topics)]
+    pub topic_id: i64,
+    #[api_model(many_to_one = users)]
+    pub author_id: i64,
 
     // TODO: custom function from topics and users joined table
     // #[api_model(summary)]
     // pub choice: Option<String>,
-    #[api_model(summary)]
-    pub nickname: String,
     #[api_model(summary, action = comment, related = String)]
     pub content: String,
 
-    #[api_model(many_to_one = topics)]
-    pub topic_id: i64,
-
-    #[api_model(summary, one_to_many = user_comments, aggregator = count)]
+    #[api_model(summary, one_to_many = comments_likes, foreign_key = comment_id, aggregator = count)]
     pub likes: u64,
-    #[api_model(many_to_many = user_comments, foreign_table_name = users, foreign_primary_key = user_id, foreign_reference_key = comment_id, aggregator = exist, exist)]
+    #[api_model(many_to_many = comments_likes, foreign_table_name = users, foreign_primary_key = user_id, foreign_reference_key = comment_id, aggregator = exist, exist, unique)]
     pub is_liked: bool,
 }
+
+#[api_model(base = "/v1/topics/:topic-id/comments", table = comment_likes, iter_type = QueryResponse)]
+pub struct CommentLike {
+    #[api_model(summary, primary_key, read_action = find_by_id)]
+    pub id: i64,
```

Comment:
```suggestion
    #[api_model(summary, primary_key)]
    pub id: i64,
```

이거는 DB  쪽 조회를 위해서 사용될거 같은데요. 여기에 추가하지 말아주세요. 
read_action, action, action_by_id 는 Client 가 호출하는 관점에서만 사용해주세요.